### DISCUSSION TITLE
Are the hitsounds in osu!mania treated as played on the middle regardless of column position?

### DISCUSSION THREAD

**Discussion by aryoadhi** [ppy/osu#23165] [Q&A]
As you all know, in osu!mania, there is a common problem where the notes position dictates the balance of left/right channel. In nutshell, notes on left louder on left, so does right louder on right. However, this seems to only happen in the stable release of the game and not in the latest version, osu!lazer.

Does this behavior occur in lazer? It would be great if it's not.

**Comment by Theighlin** [comment]
It does on standard, not sure about mania. You can try and look for the positional sound or hitsound separation slider in the settings, turn it up and see if it does something. (Don't remember the proper name right now)

**Reply by aryoadhi** [reply]
Cool, it really does something. Though it only applies globally rather than per-mode.

What does the setting actually do? Does it merge the hitsounds to mono the closer it is to 0% or make it invariant based on lane? There are some edge cases I want to know if it makes the hitsound mono or not with 0% setting:
- The hitsound that is originally played only on one channel (i.e when checked with Audacity, the dominant sound is on one channel)
- The hitsounds that sweeps from either channel to other channel

**Reply by peppy** [reply]
The easiest way to check would be to test yourself, by creating hitsounds which are fully panned in one or the other direction. I'd like to hope it moves all playback to the one side, but would need to test to confirm whether this is how BASS handles internally.

Please do report back if you end up testing this.
